lib/user-interface/react-app/src/components/chatbot/chat-input-panel.tsx

```diff
@@ -263,7 +263,7 @@ export default function ChatInputPanel(props: ChatInputPanelProps) {
         setReadyState(ReadyState.CLOSED);
       }
     })();
-  }, [appContext, state.modelsStatus, props]);
+  }, [appContext, props.session.id]); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
`props` was causing a new refresh on render. Instead it should refresh on a new session.